data-browser/src/main/scala/hmda/dataBrowser/services/S3FileService.scala

```diff
@@ -26,6 +26,8 @@ class S3FileService(implicit mat: Materializer) extends FileService with Setting
       val content = formName(queries)
       s"${content}${fileEnding(delimiter)}"
     }
+    log.info("Persisting S3 Key: " + key)
```

Comment:
Can you remove these debugging statements @omnipresent07

Reply:
@PatrickGoRaft Changed to debug statements